lib/src/bool_extensions.dart

```diff
@@ -6,4 +6,9 @@ extension BoolExtensions on bool {
   ///
   /// `since 0.7.0`
   bool get isFalse => this == false;
+
+  /// Returns `true` when [this] is `true`
+  ///
+  /// `since 0.7.0`
+  bool get isTrue => this == true;
```

Comment:
What's the point of `isTrue`? If it's being used in a condition (like and `if`), there will be no need to add `.isTrue`.
```dart
if(true.isTrue)
```

Reply:
There is none, apart from consistency. I don't know when you would call `true.isTrue` or `false.isFalse`

`if (x.isTrue || y.isFalse) {}`